#! /bin/sh /usr/share/dpatch/dpatch-run
## 23_bls-v3opts-and-consistency.dpatch
#
# Description: adds missing and new options in the bls utility's 
#  manpage. Introduces better consistency across the different Bacula 
#  manpages.
# Origin: vendor, 
#  http://bugs.debian.org/cgi-bin/bugreport.cgi?msg=45;filename=manpage-bls-v3-missing-consistency-opts-patch;att=3;bug=499619
# Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=499619
# Forwarded: http://sourceforge.net/mailarchive/message.php?msg_name=200912021125.44398.kern%40sibbald.com
# Author: Lucas B. Cohen <lbc@members.fsf.org>
# Reviewed-by: Kern Sibbald <kern@sibbald.com>
# Last-Update: 2009.12.3
#
# 2009.12.7 : tweaked patch to a patchlevel 2, to accomodate dpatch and git
#
# created by Lucas B. Cohen <lbc@members.fsf.org> with dpatch v2.0.31

@DPATCH@
2
diff -urNad a/bacula-3.0.3/manpages/bls.8 b/bacula-3.0.3/manpages/bls.8
--- a/bacula-3.0.3/manpages/bls.8	2009-10-18 11:10:16.000000000 +0200
+++ b/bacula-3.0.3/manpages/bls.8	2009-12-03 07:27:23.000000000 +0100
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH BLS 8 "26 May 2006" "Kern Sibbald" "Network backup, recovery and verification"
+.TH BLS 8 "24 November 2009" "Kern Sibbald" "Network backup, recovery and verification"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -45,26 +45,29 @@
 .BI \-d\  nn
 Set debug level to \fInn\fP.
 .TP
+.BI \-dt
+Print timestamp in debug output
+.TP
 .BI \-e\ <file>
-Specify exclude list file
+Specify exclude list file.
 .TP
 .BI \-i\  <file>
-Specify include list file
+Specify include list file.
 .TP
 .BI \-j
-List jobs
+List jobs.
 .TP
 .BI \-k
-List blocks
+List blocks.
 .TP 
 .I (no \-j or \-k option)
 List saved files
 .TP
 .BI -L
-Dump label
+Dump label.
 .TP
 .BI \-p
-Proceed despite errors
+Proceed inspite of errors.
 .TP
 .BI \-V\  volumes
 Specify volume names (separated by '|').
